code/datums/uplink/ammunition.dm

```diff
@@ -99,37 +99,37 @@
 /datum/uplink_item/item/ammo/ihclrifle
 	name = ".25 caseless magazine"
 	desc = "Standard .25 magazine with lethal ammunition. Used mostly in IHS carabines. Can fit 30 bullets."
-	item_cost = 2
+	item_cost = 3
 	path = /obj/item/ammo_magazine/ihclrifle
 
 /datum/uplink_item/item/ammo/ihclrifle/highvelocity
 	name = ".25 caseless HV magazine"
 	desc = "Standard .25 magazine with high velocity ammunition. Used mostly in IHS carabines. Can fit 30 bullets."
-	item_cost = 3
+	item_cost = 4
 	path = /obj/item/ammo_magazine/ihclrifle/hv
 
 /datum/uplink_item/item/ammo/cspistol
 	name = ".25 caseless pistol magazine"
 	desc = "Pistol .25 magazine with lethal ammunition. Used solely in Mandella. Can fit 10 bullets."
-	item_cost = 1
+	item_cost = 2
 	path = /obj/item/ammo_magazine/cspistol
 
 /datum/uplink_item/item/ammo/cspistol/highvelocity
 	name = ".25 caseless HV pistol magazine"
 	desc = "Pistol .25 magazine with high-velocity ammunition. Used solely in Mandella. Can fit 10 bullets."
-	item_cost = 3
+	item_cost = 2
 	path = /obj/item/ammo_magazine/cspistol/hv
 
 /datum/uplink_item/item/ammo/lrifle
 	name = ".30 Rifle magazine"
 	desc = "Long .30 magazine with lethal ammunition. Can fit 30 bullets."
-	item_cost = 3
+	item_cost = 1
 	path = /obj/item/ammo_magazine/lrifle
 
 /datum/uplink_item/item/ammo/lrifle/highvelocity
 	name = ".30 Rifle HV magazine"
 	desc = "Long .30 magazine with high velocity ammunition. Can fit 30 bullets."
-	item_cost = 4
+	item_cost = 2
 	path = /obj/item/ammo_magazine/lrifle/highvelocity
```

Comment:
Those changes were for .30 strips for rifles. Let them stay at 3 and 4.

Reply:
you mean the small ammo boxes? becouse i cant find .30 strips in the uplink